server/src/metadata.rs

```diff
@@ -48,6 +48,26 @@ pub mod entities_metadata_query {
         entities: Vec<EntityMetadata>,
     }
 
+    #[derive(Serialize)]
+    enum CreatorType {
+        Person,
```

Comment:
I have the feeling that an enum for just one value is not needed here (since currently this enum is only used at one place)... However it's not a big deal to leave the enum here, I do not have strong options.